### DISCUSSION TITLE
Capturing payment on the backend side

### DISCUSSION THREAD

**Discussion by pawelpetruch** [flutter-stripe/flutter_stripe#384] [Q&A]
**Is your feature request related to a problem? (please describe)**
I would like to capture the payment from a PaymentIntent on the backend side, whilst using PaymentSheet (for convenience).
How can I retrieve the selected PaymentMethod from the PaymentSheet?
Does GooglePay and ApplePay support confirming a PaymentIntent on the backend side?

**Describe the solution you'd like**
I'd like to be able to get the selected PaymentMethod from the PaymentSheet but not confirm it via confirmPaymentSheetPayment. Then, I could send this PaymentMethod to the backend where it would try to authorise this method, process some data and if the data has been processed successfully - charge the total amount.

**Comment by huangkaichang** [comment]
Same question here. Need to get the PaymentMethod, create a PaymentIntent and capture later.
